### PR TITLE
Magiclysm - Introduces Misc Magical Items

### PR DESCRIPTION
**Pull request by foulman**
#### Summary
```SUMMARY: Mods "Magiclysm - Misc magic items"```

#### Purpose of change
Introduces tokens, coins that perform a one-way transformation into the depicted object, used for emergencies, as well as several other miscellaneous magical items that will be found in Magiclysm locations such as the Magic Shop and Magic Basement.

#### Testing
All the items were tested and performed as expected.

#### Additional context
Nothing will spawn until itemgroup PR.
5/6

### CODE REVIEW

File: data/mods/Magiclysm/enchanted/enchanted_tokens.json [hunk 1]
```diff
@@ -0,0 +1,179 @@
+[
+  {
+    "abstract": "mtoken",
+    "name": "magic token",
+    "type": "GENERIC",
+    "category": "other",
+    "weight": "8 g",
+    "volume": 0,
```
Comment (KorGgenT) [inline data/mods/Magiclysm/enchanted/enchanted_tokens.json]:
```suggestion
    "volume": "1 ml",
```
an american dollar coin is about 1.102 mL, [see this chart](https://www.usmint.gov/learn/coin-and-medal-programs/coin-specifications)

File: data/mods/Magiclysm/enchanted/enchanted_misc.json [hunk 2]
```diff
@@ -0,0 +1,189 @@
+[
+  {
+    "type": "TOOL",
+    "id": "heat_cube",
+    "symbol": "#",
+    "color": "light_gray",
+    "name": "heat cube",
+    "description": "A smooth steel cube the size of your fist. Several buttons on the sides activate the powers of the cube.",
+    "price": 20000,
+    "price_postapoc": 1000,
```
[the remark this answers is not in the document]
Comment (foulman) [reply data/mods/Magiclysm/enchanted/enchanted_misc.json]: hah, I'm sure I copied that from somewhere, probably the hotplate stats. I'll up it.

File: data/mods/Magiclysm/enchanted/enchanted_tokens.json (hunk 1 above)
Comment (foulman) [reply data/mods/Magiclysm/enchanted/enchanted_tokens.json]: Ah, good. I copied this from the existing coin item in the game.

File: data/mods/Magiclysm/enchanted/enchanted_misc.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (foulman) [reply data/mods/Magiclysm/enchanted/enchanted_misc.json]: That should be better.

File: data/mods/Magiclysm/enchanted/enchanted_tokens.json [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+      "active": false,
+      "moves": 200,
+      "msg": "You say the command word engraved on the token, and it rapidly grows and morphs into a shiny pristine mace!"
+    }
+  },
+  {
+    "copy-from": "mtoken",
+    "type": "GENERIC",
+    "id": "mtoken_q_staff",
+    "name": "pike token",
```
Comment (KorGgenT) [inline data/mods/Magiclysm/enchanted/enchanted_tokens.json]: quarterstaff token

File: data/mods/Magiclysm/enchanted/enchanted_tokens.json [hunk 4]
```diff
@@ -0,0 +1,179 @@
...
+      "moves": 200,
+      "msg": "You say the command word engraved on the token, and it rapidly grows and morphs into a shiny pristine toolbox!"
+    }
+  },
+  {
+    "copy-from": "mtoken",
+    "type": "GENERIC",
+    "id": "mtoken_crowbar",
+    "name": "crowbar token",
+    "name_plural": "crowbar tokens",
```
Comment (KorGgenT) [inline data/mods/Magiclysm/enchanted/enchanted_tokens.json]:
```suggestion
```

File: data/mods/Magiclysm/enchanted/enchanted_tokens.json [hunk 5]
```diff
@@ -0,0 +1,179 @@
...
+      "moves": 200,
+      "msg": "You say the command word engraved on the token, and it rapidly grows and morphs into a shiny pristine screwdriver set!"
+    }
+  },
+  {
+    "copy-from": "mtoken",
+    "type": "GENERIC",
+    "id": "mtoken_toolbox",
+    "name": "toolbox token",
+    "name_plural": "toolbox tokens",
```
Comment (KorGgenT) [inline data/mods/Magiclysm/enchanted/enchanted_tokens.json]:
```suggestion
```

File: data/mods/Magiclysm/enchanted/enchanted_tokens.json [hunk 6]
```diff
@@ -0,0 +1,179 @@
...
+      "moves": 200,
+      "msg": "You say the command word engraved on the token, and it rapidly grows and morphs into a shiny pristine hammer!"
+    }
+  },
+  {
+    "copy-from": "mtoken",
+    "type": "GENERIC",
+    "id": "mtoken_screwdriver_set",
+    "name": "screwdriver set token",
+    "name_plural": "screwdriver set tokens",
```
Comment (KorGgenT) [inline data/mods/Magiclysm/enchanted/enchanted_tokens.json]:
```suggestion
```

File: data/mods/Magiclysm/enchanted/enchanted_tokens.json [hunk 7]
```diff
@@ -0,0 +1,179 @@
...
+      "moves": 200,
+      "msg": "You say the command word engraved on the token, and it rapidly grows and morphs into a pristine quarterstaff!"
+    }
+  },
+  {
+    "copy-from": "mtoken",
+    "type": "GENERIC",
+    "id": "mtoken_hammer",
+    "name": "hammer token",
+    "name_plural": "hammer tokens",
```
Comment (KorGgenT) [inline data/mods/Magiclysm/enchanted/enchanted_tokens.json]:
```suggestion
```

File: data/mods/Magiclysm/enchanted/enchanted_tokens.json [hunk 8]
```diff
@@ -0,0 +1,179 @@
...
+      "moves": 200,
+      "msg": "You say the command word engraved on the token, and it rapidly grows and morphs into a shiny pristine mace!"
+    }
+  },
+  {
+    "copy-from": "mtoken",
+    "type": "GENERIC",
+    "id": "mtoken_q_staff",
+    "name": "pike token",
+    "name_plural": "pike tokens",
```
Comment (KorGgenT) [inline data/mods/Magiclysm/enchanted/enchanted_tokens.json]:
```suggestion
```

File: data/mods/Magiclysm/enchanted/enchanted_tokens.json [hunk 9]
```diff
@@ -0,0 +1,179 @@
...
+      "moves": 200,
+      "msg": "You say the command word engraved on the token, and it rapidly grows and morphs into a shiny pristine pike!"
+    }
+  },
+  {
+    "copy-from": "mtoken",
+    "type": "GENERIC",
+    "id": "mtoken_mace",
+    "name": "mace token",
+    "name_plural": "mace tokens",
```
Comment (KorGgenT) [inline data/mods/Magiclysm/enchanted/enchanted_tokens.json]:
```suggestion
```

File: data/mods/Magiclysm/enchanted/enchanted_tokens.json [hunk 10]
```diff
@@ -0,0 +1,179 @@
...
+      "moves": 200,
+      "msg": "You say the command word engraved on the token, and it rapidly grows and morphs into a shiny pristine battle axe!"
+    }
+  },
+  {
+    "copy-from": "mtoken",
+    "type": "GENERIC",
+    "id": "mtoken_pike",
+    "name": "pike token",
+    "name_plural": "pike tokens",
```
Comment (KorGgenT) [inline data/mods/Magiclysm/enchanted/enchanted_tokens.json]:
```suggestion
```

File: data/mods/Magiclysm/enchanted/enchanted_tokens.json [hunk 11]
```diff
@@ -0,0 +1,179 @@
...
+      "moves": 200,
+      "msg": "You say the command word engraved on the token, and it rapidly grows and morphs into a shiny pristine broadsword!"
+    }
+  },
+  {
+    "copy-from": "mtoken",
+    "type": "GENERIC",
+    "id": "mtoken_battleaxe",
+    "name": "battleaxe token",
+    "name_plural": "battle axe tokens",
```
Comment (KorGgenT) [inline data/mods/Magiclysm/enchanted/enchanted_tokens.json]:
```suggestion
```

File: data/mods/Magiclysm/enchanted/enchanted_tokens.json [hunk 12]
```diff
@@ -0,0 +1,179 @@
...
+      "moves": 200,
+      "msg": "You say the command word engraved on the token, and it rapidly grows and morphs into a shiny pristine arming sword!"
+    }
+  },
+  {
+    "copy-from": "mtoken",
+    "type": "GENERIC",
+    "id": "mtoken_broadsword",
+    "name": "broadsword token",
+    "name_plural": "broadsword tokens",
```
Comment (KorGgenT) [inline data/mods/Magiclysm/enchanted/enchanted_tokens.json]:
```suggestion
```

File: data/mods/Magiclysm/enchanted/enchanted_tokens.json [hunk 13]
```diff
@@ -0,0 +1,179 @@
...
+      "moves": 200,
+      "msg": "You say the command word engraved on the token, and it rapidly grows and morphs into a shiny pristine longsword!"
+    }
+  },
+  {
+    "copy-from": "mtoken",
+    "type": "GENERIC",
+    "id": "mtoken_arming_sword",
+    "name": "arming sword token",
+    "name_plural": "arming sword tokens",
```
Comment (KorGgenT) [inline data/mods/Magiclysm/enchanted/enchanted_tokens.json]:
```suggestion
```

File: data/mods/Magiclysm/enchanted/enchanted_tokens.json [hunk 14]
```diff
@@ -0,0 +1,179 @@
...
+    "material": [ "silver" ],
+    "symbol": "o",
+    "color": "light_gray"
+  },
+  {
+    "copy-from": "mtoken",
+    "type": "GENERIC",
+    "id": "mtoken_longsword",
+    "name": "longsword token",
+    "name_plural": "longsword tokens",
```
Comment (KorGgenT) [inline data/mods/Magiclysm/enchanted/enchanted_tokens.json]:
```suggestion
```

**Pull request merged**